test/esinstall/config-package-svelte/config-package-svelte.test.js

```diff
@@ -0,0 +1,27 @@
+const {install} = require('../../../esinstall/lib');
+
+require('jest-specific-snapshot'); // allows to call expect().toMatchSpecificSnapshot(filename, snapshotName)
+
+describe('config-package-svelte', () => {
+  it('fails when no svelte plugin provided', async () => {
+    return expect(() =>
+      install(['simple-svelte-autocomplete'], {cwd: __dirname, packageLookupFields: ['svelte']}),
+    ).rejects.toThrowError(`Install failed.`);
+    // TODO:
+    // Assert the reason: Failed to load ../../../node_modules/simple-svelte-autocomplete/src/SimpleAutocomplete.svelte
+    // Try installing rollup-plugin-svelte and adding it to Snowpack (https://www.snowpack.dev/#custom-rollup-plugins)
```

Comment:
👍🏻 

Should we add the reason in this PR? Or does that require a bigger change?

Reply:
I could have made this more clear: that is currently what running esinstall will log, but it does so in a separate console.error call instead of the actual throw error. This is that issue i'd mentioned at standup today.

I think my vote would be to add these currently-console-error'd messages as a `error.reason` property, but we can tackle in a future PR